docs/amazon.aws.s3_bucket_module.rst

```diff
@@ -639,7 +639,11 @@ Examples
         state: present
         delete_public_access: true
 
-
+    # Delete a bucket policy from bucket
```

Comment:
Could you please add this example to the module inside the  EXAMPLES block?

Reply:
Oh, I also managed to leave that one out. Also managed to mangle my fork's git history while fixing this (as you can see from the changes :D ). Just like a real professional.

Anyways, I added this and cleaned up the changes in generated code. Sorry for the mess!